src/main/kotlin/org/rust/ide/intentions/MoveTypeConstraintToParameterListIntention.kt

```diff
@@ -55,6 +56,10 @@ class MoveTypeConstraintToParameterListIntention : RsElementBaseIntentionAction<
             if (bs.isNotEmpty()) {
                 append(bs.joinToString(separator = "+", prefix = ":") { it.text })
             }
+            if (param is RsTypeParameter && param.typeReference != null) {
+                val default = (param as? RsTypeParameter)?.typeReference?.text
+                if (default != null) append("=$default")
+            }
```

Comment:
You don't need the `if`. You can replace
```kotlin
if (param is RsTypeParameter && param.typeReference != null) {
    val default = (param as? RsTypeParameter)?.typeReference?.text
    if (default != null) append("=$default")
}
```
with just
```kotlin
val default = (param as? RsTypeParameter)?.typeReference?.text
if (default != null) append("=$default")
```

Reply:
My bad. Fixed.